### DISCUSSION TITLE
How to Add extra layer for allure cucumber jvm

### DISCUSSION THREAD

**Discussion by 301yutangLin** [allure-framework/allure2#1788] [Questions & Support] [answered]
Hi, I am using allure cucumber jvm in my maven project. I wanted to add an extra hierarchy in the allure report

In example.feature file, i have following:

Feature: Testing
   Scenario: name of the scenario

How do I add an extra hierarchy for this (tag?), so in allure report it can look something like:
Extra layer -> Testing -> name of the scenario

**Comment by baev** [comment, marked as answer]
To organize your tests by an extra layer, you can use the Allure Runtime API. This is accomplished by including 
`Allure.epic("Extra Layer")` in your step body.

For categorizing all your tests in this manner, consider employing the *Labels via Properties* feature; see [the docs](https://allurereport.org/docs/cucumberjvm-configuration/) for the details.

Further information can be found in the Allure Cucumber JVM documentation, available at [Allure Report](https://allurereport.org/docs/cucumberjvm/).

**Reply by david-m-globant** [reply]
Hi, how are you? You know, the solution offered does not work for me `Allure.epic("Extra Layer")`, could it be that there is some incompatibility with pytest-bdd? Could you share an example where using pytest-bdd the extra layer is added? thanks
